docs/application/web/guides/machine-learning/single.md

```diff
@@ -0,0 +1,160 @@
+# Single
+
+You can use the MLSingleShot API to load the existing neural network model from the storage. After loading the model, you can invoke it with a single instance of input data. Then, you can get the inference output result.
+
+The main features of the MLSingleShot API include:
+
+- Loading a neural network model
+
+  You can [open the neural network model](#open-the-model) from storage and configure a runtime environment.
+
+- Invoking the neural network model
+
+  After creating the SingleShot instance, you can [invoke the model](#invoke) with the input data and get the inference output result.
+
+- Fetching the inference result
+
+  You can [fetch the inference result](#fetch-the-result) after invoking the respective model.
+
+## Prerequisites
+
+To access files using the MLSingleShot API (in [mobile](../../api/latest/device_api/mobile/tizen/ml_single.html), [wearable](../../api/latest/device_api/wearable/tizen/ml_single.html), and [TV](../../api/latest/device_api/tv/tizen/ml_single.html) applications), the application has to request storage privileges by adding the following privileges to the `config.xml` file:
+
+```xml
+<!-- for accessing internal storage only -->
+<tizen:privilege name="http://tizen.org/privilege/mediastorage"/>
+<!-- for accessing external storage only -->
+<tizen:privilege name="http://tizen.org/privilege/externalstorage"/>
+```
+
+Additionally, to access files using the MLSingleShot API (in [mobile](../../api/latest/device_api/mobile/tizen/ml_single.html) and [wearable](../../api/latest/device_api/wearable/tizen/ml_single.html) applications), the application has to request [proper storage permissions](../security/privacy-related-permissions.md) using the PPM API (in [mobile](../../api/latest/device_api/mobile/tizen/ppm.html) and [wearable](../../api/latest/device_api/wearable/tizen/ppm.html) applications).
+
+## Open the model
+
+1. To load the model from storage, use `openModel` method:
```

Comment:
https://github.com/Samsung/tizen-docs/pull/1408#discussion_r582918923
annie-samsung
We don't use the term method or function in the guides hence, request you to remove it.

```suggestion
1. To load the model from storage, use `openModel()`:
```
also check whole file